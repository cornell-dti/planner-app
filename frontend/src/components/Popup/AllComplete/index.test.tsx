import React from 'react';
import renderer from 'react-test-renderer';
import { AllComplete } from '.';

const createTest = (completedTasksCount: number, allTasksCount: number): void => {
  it(`AllComplete(${completedTasksCount}, ${allTasksCount}) matches snapshot`, () => {
    const tree = renderer.create(
      <AllComplete completedTasksCount={completedTasksCount} allTasksCount={allTasksCount} />,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
};

createTest(0, 3);
createTest(1, 3);
createTest(2, 3);
createTest(3, 3);
