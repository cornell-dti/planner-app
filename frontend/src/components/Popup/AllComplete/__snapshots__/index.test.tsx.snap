// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllComplete(0, 3) matches snapshot 1`] = `
<div
  className="Main Hidden"
>
  <span
    className="ConfWrap"
  >
    <div
      style={
        Object {
          "position": "relative",
        }
      }
    />
  </span>
</div>
`;

exports[`AllComplete(1, 3) matches snapshot 1`] = `
<div
  className="Main Hidden"
>
  <span
    className="ConfWrap"
  >
    <div
      style={
        Object {
          "position": "relative",
        }
      }
    />
  </span>
</div>
`;

exports[`AllComplete(2, 3) matches snapshot 1`] = `
<div
  className="Main Hidden"
>
  <span
    className="ConfWrap"
  >
    <div
      style={
        Object {
          "position": "relative",
        }
      }
    />
  </span>
</div>
`;

exports[`AllComplete(3, 3) matches snapshot 1`] = `null`;
